pkg/util/testingjobs/paddlejob/wrappers.go

```diff
@@ -83,21 +89,20 @@ func MakePaddleJob(name, ns string) *PaddleJobWrapper {
 }
 
 // PriorityClass updates job priorityclass.
-func (j *PaddleJobWrapper) PriorityClass(pc string) *PaddleJobWrapper {
```

Comment:
I think it is preferred to keep builder functions to return ` *PaddleJobWrapper`.

 I would suggest to follow the pattern we have for MPIJob and other implementations of GenericJob.

I think we could declare the alias type 
`type PaddleJob kftraining.PaddleJob`, and extend the alias with the `Object`, `RunPolicy`, `ReplicaSpecs`, and `JobStatus` functions, so that it implements `Kubeflowjob` as above. Then, if the wrapper embeds the extended type `type PaddleJobWrapper struct{ PaddleJob }` , then you should be able to implement the `Obj` by just returning the &j.PaddleJob.

Reply:
Yes, I see that PaddleJobWrapper is a builder, so I shouldn't modify it. I didn't create a new type PaddleJob, because in that case I'll need to change existing unit test. For first integration test seems I don't need a special wrapper. I'll probably need something for other integration tests, but I would like to split it to another PR.